configshell/__init__.py

```diff
@@ -25,3 +25,8 @@
 from .node import ConfigNode, ExecutionError
 from .prefs import Prefs
 from .shell import ConfigShell
+
+__version__ = '1.1.25'
+__url__ = 'http://github.com/open-iscsi/configshell-fb'
+__description__ = 'A framework to implement simple but nice CLIs.',
```

Comment:
Is ',' needed here ?

Reply:
good catch Xiubo, will fit it now. Thanks!